models/cnn/cnn_backbone.py

```diff
@@ -0,0 +1,49 @@
+import torch.nn as nn
+import torch.nn.functional as F
+
+from .cnn import CNN
+
+class CNN(nn.Module):
```

Comment:
What is this backbone exactly for? Please add a doc/comment and please explain how this is going to be integrated into the pipeline.

Reply:
I see its partly explained in the model readme. But I actually think that the backbone models are not really needed as the backbones are used in the main_model file.

What would be good is to add the option tho the DL model to have a decoder activated or not. This is needed for  Masked prediction where every model that should be applicable to MP must have an autoencoder mode and an encoder only mode.

Reply:
The backbone files are needed, because different things have to be called depending on the model. For example the transformer requires the positional embeddings, the CNN (if we are going to replicate it as the paper does) uses a mlp after the CNN for the CRL. 

However, the backbone files are still rough drafts and can be changed/removed if we aim to do different things with the model

Reply:
Alright, lets keep them for now. We can change as needed